.github/workflows/docs.yml

```diff
@@ -39,21 +40,25 @@ jobs:
         shell: bash
         run: |
           tox -edocs
-      - name: Prepare docs artifact
-        if: always()
-        shell: bash
-        run: |
-          mkdir artifact
-          cp -a docs/_build/html artifact/addon_utils_html_docs
       - name: Upload docs artifact
         if: always()
-        uses: actions/upload-artifact@v4
-        with:
-          name: addon_utils_html_docs
-          path: ./artifact
-      - name: Deploy docs
-        if: ${{ github.ref == 'refs/heads/stable/0.1' }}
-        uses: peaceiris/actions-gh-pages@v4
+        uses: actions/upload-pages-artifact@v3
         with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          publish_dir: ./docs/_build/html/
+          name: qiskit-addon-utils-htmldocs
```

Comment:
In #1 I was not setting a name. Not sure if this affects what happens below but I assume it does not.